projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -147,19 +148,20 @@ public function initial_state() {
 
 		$shares_count = $publicize->get_publicize_shares_count( Jetpack_Options::get_option( 'id' ) );
```

Comment:
I think you might be about to hate me! If we adapt `get_publicize_shares_count` to call the new endpoint, then we probably don't need the `Shares` class. We might want to rename that method, or create a new one and have this one call it so we don't break the API.